public/angular/js/controllers/email.js

```diff
@@ -0,0 +1,51 @@
+angular.module('Aggie')
+
+.controller('EmailSettingsController', [
+  '$scope',
+  'Settings',
+  '$timeout',
+  '$filter',
+  'FlashService',
+  '$modal',
+  function($scope, Settings, $timeout, $filter, flash, $modal) {
+
+    $scope.mail = '';
+    $scope.transport = {};
+    Settings.get('email', function success(data) {
+      $scope.email = data.email.from;
+      $scope.transport = angular.copy(data.email.transport);
+    }, failure);
+
+    function success() {
+      flash.setNoticeNow('The mail setting has been correctly saved');
+    }
+
+    $scope.saveEmail = function() {
```

Comment:
I fear this is not a very scalable approach to handling settings. You will have one method for each saveable setting. Perhaps fine for now but you will want to refactor once more are added.